IBAnimatable/GradientDesignable.swift

```diff
@@ -89,7 +89,7 @@ private class PrivateGradientView: UIView {
 
   init(frame: CGRect, layer: CAGradientLayer) {
     super.init(frame: frame)
-    layer.insertSublayer(layer, at: 0)
+    self.layer.insertSublayer(layer, at: 0)
```

Comment:
next time, we may use `gradientLayer` for the parameter instead of the `layer` which is same as the property.